### ISSUE TITLE
Facebook/LinkedinIcon not rendering on the DOM

### ISSUE THREAD

**Issue by rimatla** [joashp/material-design-template#12]
Both of these (i tags) are not rendering the Facebook and/or LinkedIn icons under the contact page.

index.html lines 359 and 369
class="small fa fa-facebook-square white-text"
class="fa fa-linkedin-square"

**Comment by joashp** [comment]
On the demo website. All the icons are being rendered. Check your browser console for any errors, these are all font awesome icons.

**Comment by rimatla** [comment]
You're right. Probably a cache issue w/ Chrome. 
Do you know in what file I can **direct** the emails from contact page (submit button) to my email domain?

Thanks Joash.

**Comment by joashp** [comment]
You just have to change the email id in contact file.
https://github.com/joashp/material-design-template/blob/master/www/contact.php#L5

The email will be sent to your inbox.

**Comment by rimatla** [comment]
Hi Joash,

Thank you for your insight. I'm a JS guy and know nothing of PHP. I passed the code bellow through a PHP checker and got no errors. Yet every time I try testing on the web page I get an error after pressing the Submit button

`<?php

if(isset($_POST['email'])) {

```
$email_to = "test@gmail.com";

$email_subject = "Website Contact";


function died($error) {

    // your error code can go here

    echo "We are very sorry, but there were error(s) found with the form you submitted. ";

    echo "These errors appear below.<br /><br />";

    echo $error."<br /><br />";

    echo "Please go back and fix these errors.<br /><br />";

    die();

}



// validation expected data exists

if(!isset($_POST['name']) ||

    !isset($_POST['email']) ||

    !isset($_POST['message'])) {

    died('We are sorry, but there appears to be a problem with the form you submitted.');

}


$name = $_POST['name'];
$email = $_POST['email'];
$message = $_POST['message'];


$email_message = "Form details below.\n\n";



function clean_string($string) {

  $bad = array("content-type","bcc:","to:","cc:","href");

  return str_replace($bad,"",$string);

}



$email_message .= "Name: ".clean_string($name)."\n";

$email_message .= "Email: ".clean_string($email)."\n";

$email_message .= "Message: ".clean_string($message)."\n";


// create email headers

$headers = 'From: '.$email."\r\n".

'Reply-To: '.$email."\r\n" .

'X-Mailer: PHP/' . phpversion();

@mail($email_to, $email_subject, $email_message, $headers);
```

}

echo "Thank you for contacting me. Will be in touch with you very soon.";
?>
`

**Comment by joashp** [comment]
@rimatla You gotta put an actual email address for it to work. 
And I hope you are running this on a server. As for PHP code to run, you need a server .

**Comment by rimatla** [comment]
@joashp Now you called it. Server! dang... I'm not running one...

The email was just to illustrate...(I'm actually using a real email)

On the other hand, as soon I pushed the Material Design frames to my repo. I keep constantly getting a 404 error from GitHub. I emailed GitHub and they said that they couldn't help me and that I had to contact you directly (?).

This is the github page.
rimatla.github.io

Thanks again!
